### PR TITLE
WIFI-1658 Add Channel Number Status Support

### PR DESCRIPTION
**Pull request by lynnshi-design**

### CODE REVIEW

File: opensync-ext-cloud/src/main/java/com/telecominfraproject/wlan/opensync/external/integration/OpensyncExternalIntegrationCloud.java [hunk 1]
```diff
@@ -645,6 +646,19 @@ private void updateApStatus(Equipment ce, ConnectNodeInfo connectNodeInfo) {
...
             networkAdminStatusRec = statusServiceInterface.update(networkAdminStatusRec);
+            
+            Status channelStatusRec = statusServiceInterface.getOrNull(ce.getCustomerId(), ce.getId(),
+                    StatusDataType.RADIO_CHANNEL);
+            if (channelStatusRec == null) {
+            	channelStatusRec = new Status();
+            	channelStatusRec.setCustomerId(ce.getCustomerId());
+            	channelStatusRec.setEquipmentId(ce.getId());
+            	channelStatusRec.setStatusDataType(StatusDataType.RADIO_CHANNEL);
+                EquipmentChannelStatusData channelStatusData = new EquipmentChannelStatusData();
```
Comment (dmitry-toptygin-connectus-ai) [inline opensync-ext-cloud/src/main/java/com/telecominfraproject/wlan/opensync/external/integration/OpensyncExternalIntegrationCloud.java:657]: This creates an empty status, with no channel info. 
The channels will only be populated after wifiRadioStatusDbTableUpdate - when channels are changed. 
Does it make sense to read the OpensyncAPRadioState in here and populate the channels, so there's no empty values initially?

Comment (lynnshi-design) [reply opensync-ext-cloud/src/main/java/com/telecominfraproject/wlan/opensync/external/integration/OpensyncExternalIntegrationCloud.java:657]: I will create it when it is ready. Thanks.

**Review by mikehansen1970 — DISMISSED** [review]
Approved with the changes requested by Dmitry

File: opensync-ext-cloud/src/main/java/com/telecominfraproject/wlan/opensync/external/integration/OpensyncExternalIntegrationCloud.java (hunk 1 above)
Comment (dmitry-toptygin-connectus-ai) [reply opensync-ext-cloud/src/main/java/com/telecominfraproject/wlan/opensync/external/integration/OpensyncExternalIntegrationCloud.java:657]: If it's not ready now - please create a JIRA to be addressed in a future

Comment (lynnshi-design) [reply opensync-ext-cloud/src/main/java/com/telecominfraproject/wlan/opensync/external/integration/OpensyncExternalIntegrationCloud.java]: Just made the change, now RADIO_CHANNEL status will be created only when there is data.

**Review by mikehansen1970 — APPROVED** [review]

**Pull request merged**